bin/fetch-apis.js

```diff
@@ -1,6 +1,6 @@
 const got = require('got');
 
-const MANIFEST_URL = 'http://references.taskcluster.net/manifest.json';
+const MANIFEST_URL = process.env.MANIFEST_URL;
```

Comment:
This is fine for now.  We need to revisit the generation process -- and @petemoore will be working on that soon.